lib/fastlane/actions/lcov.rb

```diff
@@ -0,0 +1,83 @@
+module Fastlane
+  module Actions
+    class LcovAction < Action
+      @@derived_data_path = "#{File.expand_path('~')}/Library/Developer/Xcode/DerivedData/"
```

Comment:
by this you meant "File.expand_path('~')" ? changed to Dir.home , i'm not sure why i didn't do like that in the first place .. =)